### DISCUSSION TITLE
Advertising campaign report field issues

### DISCUSSION THREAD

**Discussion by cyncxl300** [amzn/ads-advanced-tools-docs#86] [Q&A] [answered]
Please help me understand why the report is returning the values "legacy", "optimizeForSales", and "manual" for the "campaignBiddingStrategy" column, instead of the expected values "Dynamic bids - down only", "Fixed bids", "Dynamic bids - down only", "Dynamic bids - down only", and "Dynamic bids - up and down". The backend advertising strategy settings are correct.

**Comment by elizafarley** [comment]
Hi @cyncxl300, could you confirm which ad type (SP/SB/SD/DSP) and report type this is occurring for?

**Reply by cyncxl300** [reply]
<img width="1489" alt="微信图片_20230516002232" src="https://github.com/amzn/ads-advanced-tools-docs/assets/133635097/c6b2f7a4-bf19-4cd3-92d4-e3940d874c56">
Hello! It's about the SP campaigns report.

**Reply by elizafarley** [reply, marked as answer]
I understand your confusion. The API uses different naming conventions for bidding strategies in Sponsored Products campaigns. Take a look at this table to understand how the API values map to the UI values.

|API value | UI value | 
|--------|------|
| legacy | Dynamic bids - down only | 
| optimizeForSales | Dynamic bids - up and down |
| manual | Fixed bid |

Let me know if this answers your question. I'll also work on getting this mapping added to the metric description for `campaignBiddingStrategy`.

**Reply by cyncxl300** [reply]
I understand, thank you very much!
